src/System.Private.CoreLib/shared/System/Globalization/CultureData.cs

```diff
@@ -833,11 +831,11 @@ internal string DisplayName
             {
                 if (_sLocalizedDisplayName == null)
                 {
-                    if (this.IsSupplementalCustomCulture)
+                    if (IsSupplementalCustomCulture)
                     {
-                        if (this.IsNeutralCulture)
+                        if (IsNeutralCulture)
                         {
-                            _sLocalizedDisplayName = this.SNATIVELANGUAGE;
+                            _sLocalizedDisplayName = NativeDisplayName;
```

Comment:
did we rename SNATIVELANGUAGE to NativeDisplayName? should it be NativeLanguageName? I am asking because you renamed SENGLISHLANGUAGE to EnglishLanguageName which is not consistent